src/pydvl/influence/frameworks/twice_differentiable.py

```diff
@@ -1,11 +1,21 @@
 from abc import ABC
-from typing import Callable, Generic, List, Sequence, Tuple, TypeVar
+from dataclasses import dataclass
+from typing import Any, Callable, Dict, Generic, List, Sequence, Tuple, TypeVar
 
 TensorType = TypeVar("TensorType", bound=Sequence)
 ModelType = TypeVar("ModelType")
 DeviceType = TypeVar("DeviceType")
 
 
+@dataclass(frozen=True)
+class IhvpResult(Generic[TensorType]):
```

Comment:
Starting a class name with a capital I is in other languages marking an interface. What do you think about InverseHvpResult?